_data/glossary/statefulset.yml

```diff
@@ -0,0 +1,18 @@
+id: statefulset
+name: StatefulSet
+formerly:
+- PetSet
+related:
+- Deployment
+- Pod
+tags:
+- Storage
+- Workload
+- API Object
+short-description: |
+  Manages the deployment and scaling of a set of replica Pods, *and provides guarantees about ordering*. It does so by maintaining a *unique*, sticky identity for each of its Pods.
```

Comment:
discussed offline, to keep in sync with the existing (undocumented) standards, we will describe things in a plural way (even if their filenames and titles are singular)